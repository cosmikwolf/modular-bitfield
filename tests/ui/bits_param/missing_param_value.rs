use modular_bitfield::prelude::*;

#[bitfield(bits)]
pub struct Missing {
    sign: bool,
    value: B31,
}

#[bitfield(bits(32))]
pub struct WrongMetaType {
    sign: bool,
    value: B31,
}

fn main() {}
