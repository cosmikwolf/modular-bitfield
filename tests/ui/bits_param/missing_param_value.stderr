error: encountered unsupported #[bitfield] attribute
 --> tests/ui/bits_param/missing_param_value.rs:3:12
  |
3 | #[bitfield(bits)]
  |            ^^^^

error: encountered unsupported #[bitfield] attribute format
 --> tests/ui/bits_param/missing_param_value.rs:9:12
  |
9 | #[bitfield(bits(32))]
  |            ^^^^^^^^
